### PR TITLE
Puppeteer logs dir not exists

### PR DESCRIPTION
**Pull request by aweng98**
Fixed an issue with `fs. existsSync` usage - method fails to create dir recursively. Error thrown when `logs` dir not exists.
Also removed `home-spec.ts` flaky test `Homepage is the landing page after Sign In` because it fails intermittently. It's not a good UI test to check network api calls.

### CODE REVIEW

File: e2e/app/base-page.ts [hunk 1]
```diff
@@ -228,9 +228,7 @@ export default abstract class BasePage {
    */
   async takeScreenshot(fileName: string) {
     const dir = 'logs/screenshot';
-    if (!fs.existsSync(dir)) {
-      fs.mkdirSync(dir);
-    }
+    await fse.ensureDir(dir);
```
Comment (jaycarlton) [inline e2e/app/base-page.ts]: nit: maybe name it `SCREENSHOT_DIR`.

File: e2e/app/google-login.ts [hunk 2]
```diff
@@ -91,11 +91,13 @@ export default class GoogleLoginPage extends BasePage {
    */
   async load(): Promise<void> {
     const url = configs.uiBaseUrl + configs.loginUrlPath;
-    await this.page.goto(url, {waitUntil: ['networkidle0', 'domcontentloaded'], timeout: 30000}).catch((err) => {
+    try {
+      await this.page.goto(url, {waitUntil: ['networkidle0', 'domcontentloaded'], timeout: 30000});
+    } catch (err) {
       console.error('Google login page not found. ' + err);
-      this.takeScreenshot('GoogleLoginNotFound');
+      await this.takeScreenshot('GoogleLoginNotFound');
```
Comment (jaycarlton) [inline e2e/app/google-login.ts]: nit: maybe specify `GoogleLoginPageNotFound` to distinguish from the case of an unrecognized user.

File: e2e/app/google-login.ts [hunk 3]
```diff
@@ -91,11 +91,13 @@ export default class GoogleLoginPage extends BasePage {
    */
   async load(): Promise<void> {
     const url = configs.uiBaseUrl + configs.loginUrlPath;
-    await this.page.goto(url, {waitUntil: ['networkidle0', 'domcontentloaded'], timeout: 30000}).catch((err) => {
+    try {
+      await this.page.goto(url, {waitUntil: ['networkidle0', 'domcontentloaded'], timeout: 30000});
```
Comment (jaycarlton) [inline e2e/app/google-login.ts]: nit: Maybe wrap this into a higher-level function like `this.navigate(url)`.

The timeout seems a little generous. to my untrained eye, especially for the login page.

**Comment by aweng98** [comment]
@jaycarlton Thank you. I have made changes as per your suggestion.

**Pull request merged**